bin/reth/src/db/get.rs

```diff
@@ -0,0 +1,154 @@
+use clap::Parser;
+use eyre::WrapErr;
+use reth_db::{database::Database, table::Table, tables};
+use serde::Deserialize;
+use tracing::error;
+
+use crate::utils::DbTool;
+
+#[derive(Parser, Debug)]
+/// The arguments for the `reth db get` command
+pub struct Command {
+    /// The table name
+    ///
+    /// NOTE: The dupsort tables are not supported now.
+    #[arg()]
+    pub table: String, // TODO: Convert to enum
+
+    /// The key to get content for
+    #[arg(long, value_parser = maybe_json_value_parser)]
+    pub key: String,
```

Comment:
I would not make this a flag since it is always required

Reply:
fixed: aa3ccab7d2f0418ae88b3772ae02f83273a3b190